App.test.js

```diff
@@ -0,0 +1,58 @@
+import { render, screen, fireEvent } from '@testing-library/vue'
+
+import Button from "./src/components/common/button.vue";
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+
+test('button value on click', async () => {
+  const { getByRole } = await render(Button, {
+    propsData: {
+      type: 'button',
+      clickCall: () => { }
+    }
+  });
+
+  const button = getByRole('button')
+  await fireEvent.click(button)
```

Comment:
there is no test case here. so this is wrong. 
there should be some expect statement